### ISSUE TITLE
Uncaught TypeError: Cannot read property 'exp' of undefined

### ISSUE THREAD

**Issue by iegik** [angular/batarang#253]
https://github.com/angular/angular-hint/blob/master/src/modules/scopes.js#L301

https://github.com/angular/batarang/blob/master/dist/hint.js#L2092
```
function humanReadableWatchExpression (fn) {
  //  if(!fn) {
  //      return 'undefined';
  //  }
  if (fn.exp) {
    fn = fn.exp;
  } else if (fn.name) {
    fn = fn.name;
  }
  return fn.toString();
}
```

**Comment by gary-b** [comment]
Hi,
The recently committed null/undefined check in angular hint resolves this issue. Batarang hasn't been released again since and its dist.js file is out of date. I suspect the next release will happen shortly. 
PS i can see why you went with the string 'undefined' here instead of null, null gave me no problems when i tested it if that makes it any better :)

**Comment by iegik** [comment]
You should correct package.json

**Comment by SomeKittens** [comment]
This should be fixed in v0.8.6
